src/tests/performance/prio_test.rs

```diff
@@ -192,6 +192,26 @@ async fn p002_t2_TRAFFIC_SAME_PRIO_latency() {
     run_traffic_test(high_traffic_factory, normal_traffic_factory).await;
 }
 
+#[cfg_attr(not(feature = "performance"), ignore)]
+#[tokio::test(flavor = "multi_thread", worker_threads = 8)]
+#[allow(non_snake_case)]
+async fn p002_t3_COMB_MSG_DIGEST_latency() {
+    // ZG-PERFORMANCE-002
+
+    let hash = vec![2u8; 32];
+    let high_traffic_factory =
+        PayloadFactory::new(Payload::MsgDigestSkip(HashDigest::from(&hash)), None);
```

Comment:
let's use a different hash for every message, payload factory can take care of this probably. (it shall increase the ram usage of the node, otherwise the node will update the same place in the hash table)

Reply:
Done.